src/components/attendee/IntersectingButton/IntersectingButton.tsx

```diff
@@ -14,7 +14,7 @@ interface IntersectingButtonProps
   transparent?: boolean;
   border?: BorderVariant;
   className?: string;
-  forwardRef: RefObject<HTMLButtonElement>;
+  forwardRef?: RefObject<HTMLButtonElement>;
```

Comment:
Plenty of unnecessary field definitions here, `IntersectingButtonProps` can just extend from `ButtonProps` or be 

```tsx
type IntersectingButtonProps = ButtonProps & { forwardRef?: RefObject<HTMLButtonElement> };
```

Not even the `onClick` is modified in this HoC so it might just be 

```tsx
return <Button {...rest} forwardRef={targetRef} />
```

[NOTE] It's OK to export/import ButtonProps from "components/attendee/Button" as long as they're needed elsewhere